src/controllers/ApiController.php

```diff
@@ -613,6 +618,44 @@ private function createExperiment(): Response
         return new JsonResponse(array('result' => 'success', 'id' => $id));
     }
 
+    /**
+     * @api {post} /templates Create template
+     * @apiName CreateTemplate
+     * @apiGroup Entity
+     * @apiExample {python} Python example
+     * import elabapy
+     * manager = elabapy.Manager(endpoint="https://elab.example.org/api/v1/", token="3148")
+     * response = manager.create_template()
+     * print(f"Created template with id {response['id']}.")
+     * @apiExample {shell} Curl example
+     * export TOKEN="3148"
+     * # create a template with default status
+     * curl -X POST -H "Authorization: $TOKEN" https://elab.example.org/api/v1/templates
+     * @apiSuccess {String} result success or error message
+     * @apiSuccess {String} id Id of the new template
+     * @apiSuccessExample {Json} Success-Response:
+     *     HTTP/2 200 OK
+     *     {
+     *       "result": "success",
+     *       "id": 42
+     *     }
+     */
+
+    /**
+     * Create a template
+     *
+     * @return Response
+     */
+    private function createTemplate(): Response
+    {
+        if ($this->Entity instanceof Database) {
```

Comment:
this check seems unneeded as there is already a check for instanceof Templates before calling this function

Reply:
This was implemented analogue to the `createExperiment` function in the same controller. It might be cleaned there as well. Fixed in 4c4dbce